Makefile

```diff
@@ -50,6 +50,28 @@ SHELL = /bin/bash
 VERSION := $(shell (cd "$(SRCDIR)" && python3 setup.py --version))
 COMMIT = $(shell (cd "$(SRCDIR)" && git rev-parse HEAD))
 
+# List of all files which trigger a rebuild of the
+# development targets and container
+
+# Directories to search
+SRC_DEPS_DIRS := assemblers devices inputs mounts osbuild runners sources stages tools
+
+# Function to search for Python files in a directory
+define find_python_files
+$(shell find $(1) -type f -exec file {} \; | grep "Python script" | grep -Eo "^[^:]+")
```

Comment:
If this will catch also stage files? They lack `.py` extension. There is also stage `org.osbuild.tar` which is python file but frequently by script is processed as a tar archive:)

Reply:
That's the only reason for this _not-so-nicce_ find-grep pipe - to get those files in consideration :-)

Reply:
Just running in the osbuild directory should show them all...
```
find . -type f -exec file {} \; | grep "Python script" | grep -Eo "^[^:]+"
```
(actually I now see that I have ".py" files twice - but that won't hurt `make` I guess)